locales/en/messages.json

```diff
@@ -3594,6 +3594,22 @@
         "message": "Master Multiplier:",
         "description": "Master tuning slider label"
     },
+    "pidTuningRollPitchRatioSlider": {
+        "message": "Roll Pitch Ratio:",
+        "description": "Roll pitch ratio slider label"
+    },
+    "pidTuningRollPitchRatioSliderHelp": {
+        "message": "Roll Pitch Ratio",
+        "description": "This needs a meaningful help text"
```

Comment:
This description is missing?

Reply:
@McGiverGim , this should be "Pitch Roll Ratio" (flipped) as most raise Pitch gains.  So as the slider goes up, the Pitch should go up, not Roll.  That said, i reckon this is controlled in BF now (firmware based)?

Reply:
Yes - these were missing originally in the pull request. I have added placeholders for now, so we can decide for each one if we need a help text, or if we should remove the tooltip.

@spatzengr: The respective simplified setting in the firmware is currently called `simplified_roll_pitch_ratio` - so if we decide to rename this then we should do it now, before any of this is released, and change it in the firmware too.

Reply:
@mikeller , I created the other slider tool tips and can give the new ones a shot.  I think we should change Roll/Pitch slider to Pitch/Roll for the reasons outlined above.  I also think D_min slider maybe should change, but that would not revise the name and is 50/50 to me on that one (could go either way).  

I actually just created a branch in my repo to take a look at doing this change, but i'm sure it would take me hours and you minutes.  So if you are willing, it would not hurt my feelings.  This was the only FW thing and the rest should be configurator items to polish up.

Reply:
@spatzengr: Hehe, one could argue that these hours would be well spent on honing your JS / git / GitHub skills. :stuck_out_tongue:
But I don't mind doing the change - we can probably combine it with the tooltip texts, so let me know when you have these, and I can do the change.